src/bucket/BucketList.cpp

```diff
@@ -82,7 +86,7 @@ BucketList::shouldMergeWithEmptyCurr(uint32_t ledger, uint32_t level)
     {
         // Round down the current ledger to when the merge was started, and
         // re-start the merge via prepare, mimicking the logic in `addBatch`
-        auto mergeStartLedger = mask(ledger, BucketList::levelHalf(level - 1));
+        auto mergeStartLedger = roundDown(ledger, BucketList::mask(level - 1));
```

Comment:
something went wrong with the rename here - why is `levelHalf` named `mask` now? In the earlier comment, I was saying that simplest is to revert this change completely as the refactor isn't needed anymore

Reply:
Sorry, I think I messed up my vim command. Interesting that it still passed tests. While I was fixing it though, I spotted that this rename actually is useful since `BucketIndex` uses this function as well when doing page aligning index entries. I'm reverting the erroneous refactor, but I think it's best to keep the rename for `roundDown` since it's used outside of the `BucketList` now.